sdk_container/src/third_party/coreos-overlay/coreos-base/oem-ec2/files/manglefs.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+rootfs="${1}"
+
+to_delete=(
+  /etc/systemd/system/amazon-ssm-agent.service
+  /etc/systemd/system/multi-user.target.requires/coreos-metadata-sshkeys@core.service
+  /etc/amazon/ssm/
+  /etc/eks/bootstrap.sh
+)
+
+rm -rf "${to_delete[@]/#/${rootfs}}"
+
+mkdir -p "${rootfs}/usr/lib/systemd/system/amazon-ssm-agent.service.d"
+cat > "${rootfs}/usr/lib/systemd/system/amazon-ssm-agent.service.d/10-bindpaths.conf" <<-'EOF'
+[Service]
+BindPaths=/usr/share/amazon/ssm/:/etc/amazon/ssm/ /usr/share/amazon/eks/boostrap.sh:/etc/eks/bootstrap.sh
```

Comment:
Are users expected to be able run the CLI themselves and does it also read from `/etc`? (In that case we would anyway have to have the symlinks from `/etc`, or?)

Reply:
My question was more about `ssm-cli` and whether this is used by users and needs access to the files in `/etc`.